packages/block-editor/src/store/selectors.js

```diff
@@ -3068,6 +3068,12 @@ export const getBlockEditingMode = createRegistrySelector(
 					return 'contentOnly';
 				}
 
+				// All controlling blocks are treated as content only
+				// by default.
+				if ( areInnerBlocksControlled( state, clientId ) ) {
+					return 'contentOnly';
+				}
```

Comment:
For me when Zoomed Out I can't access the inner blocks of sections anyway.

Reply:
Yes, that was added "preemptively" by me, but I don't have a case for it yet. Should probably remove it.